plugins/coverbrowser.koplugin/main.lua

```diff
@@ -41,6 +44,7 @@ local init_done = false
 local filemanager_display_mode = false -- not initialized yet
 local history_display_mode = false -- not initialized yet
 local series_mode = nil -- defaults to not display series
+local collection_display_mode = false -- not initialized yet
```

Comment:
Just swap these 2. `series_mode` is a bit unrelated to the now 3 real display modes.